### DISCUSSION TITLE
Поддержка электрических счётчиков

### DISCUSSION THREAD

**Discussion by dontsovcmc** [dontsovcmc/waterius#290] [Ideas]
Здесь давайте обсуждать поддержку эл. счётчиков, чтобы не засорять issue.
Ссылки на обсуждения:
https://github.com/dontsovcmc/waterius/issues/146
https://github.com/dontsovcmc/waterius/issues/66

**Comment by Ankarii** [comment]
А что-то планируется в этом направлении? У меня, например счетчик STAR 104/1 R1-5(60)Э 4ШИО IEK, у него выход RS-485, вообще, под верхней крышкой, а пломбируют ту что снизу, где подключение силовых проводов.

**Reply by dontsovcmc** [reply]
Напишите в чат waterius_forum, там сейчас один из пользователей делает универсальное устройство для 485 счетчиков. Как раз у вас есть шанс получить образец и проверить!

**Comment by dontsovcmc** [comment]
https://t.me/waterius_forum/29308
